l10n/en/firefox/browsers/compare/shared.ftl

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/compare/
+
+compare-shared-compare-firefox = Compare { -brand-name-firefox }
+compare-shared-footer-cta-title = Do more with { -brand-name-firefox }
+compare-shared-footer-cta-desc = Get the fast, lightweight, privacy-focused browser that’s backed by a non-profit and works across all your devices.
+compare-shared-footer-cta-button = Download { -brand-name-firefox }
+compare-shared-we-also-offer-easy = We also offer easy-to-use features such as:
+compare-shared-firefox-also-has-built-in = { -brand-name-firefox } also has built-in tools such as:
+
+# Variables
+# $pdf (url) - link to https://www.mozilla.org/firefox/features/pdf-editor/
+compare-shared-edit-pdfs-on-the-go = <a { $pdf }>Edit PDFs</a> on the go within your { -brand-name-firefox } browser window - no extra software needed.
+
+# Variables
+# $translate (url) - link to https://www.mozilla.org/firefox/features/translate/
+compare-shared-translate-a-web-page = <a { $translate }>Translate a web page</a> locally and privately.
+compare-shared-its-easy-to-switch = It’s Easy to Switch
```

Comment:
**Suggestion:** Most (if not all) of our titles across the website only have the first word capitalised (you can see some examples in the [features base page](https://www.mozilla.org/en-US/firefox/features/)). Should this be changed to `It's easy to switch` instead?